install_config/router/default_haproxy_router.adoc

```diff
@@ -166,6 +168,50 @@ $ oc adm router --dry-run -o yaml --service-account=router
 ----
 endif::[]
 
+[[deploy-router-access-logs]]
+Configuring the Router to Forward Access Logs::
+
+// tag::accesslogs[]
+You can configure {product-title} to send socket messages, or access logs, 
+to your EFK stack. The `--extended-logging` parameter appends an rsyslog 
+sidecar container to forward audit logs to standard output. 
+
+----
+$ oc adm router --extended-logging=true
+----
+
+For example, the following is a router configuration created using `--extended-logging=true`: 
+----
+# oc get pod test-router-1-xhdb9 -o yaml
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - env:
+    ....
+ - command: <1>
+   - /sbin/rsyslogd
+   - -n
+   - -i
+   - /tmp/rsyslog.pid
+   - -f
+   - /etc/rsyslog/rsyslog.conf
+   image: registry.redhat.io/openshift3/ose-haproxy-router:v3.11.188
+   imagePullPolicy: IfNotPresent
+   name: syslog
+----
+<1> The parameter added a new container to the router. In the container, 
+the rsyslog process is running as: 
+`/sbin/rsyslogd -n -i /tmp/rsyslog.pid -f /etc/rsyslog/rsyslog.conf`.
+
+Use the following command to view the access logs:
```

Comment:
See also https://docs.openshift.com/container-platform/3.11/admin_guide/router.html#viewing-logs
The default log level is warning and should be set to `info` or `debug` if you want to view the access logs.
`oc set env dc/test-router ROUTER_LOG_LEVEL=info`.